src/input/base-input.js

```diff
@@ -184,13 +184,15 @@ class BaseInput<T: EventTarget> extends React.Component<
     }[this.props.size];
     return (
       <MaskToggleButton
-        aria-label={label}
-        kind={KIND.minimal}
-        onClick={() => this.setState({isMasked: !this.state.isMasked})}
-        title={label}
-        type="button"
```

Comment:
button type defaults to `submit` so removing this may cause regressions in application forms